scripts/linters/custom_eslint_checks/rules/match-line-break.js

```diff
@@ -0,0 +1,96 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Lint check to ensure that  the line breaks between the
+ * dependencies listed in the controller of a directive or service exactly
+ * match those between the arguments of the controller function.
+ */
+
+'use strict';
+
+module.exports = {
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description: (
+        'Lint check to ensure that  the line breaks between the dependencies' +
+        ' listed in the controller of a directive or service exactly' +
+        'match those between the arguments of the controller function'),
+      category: 'Stylistic Issues',
+      recommended: true,
+    },
+    fixable: null,
+    schema: [],
+    messages: {
+      matchLineBreak: (
+        'Please ensure that the line breaks pattern between the dependencies' +
+        ' mentioned as strings and the dependencies mentioned as function' +
+        ' parameters for the corresponding controller should exactly match.')
+    },
+  },
+
+  create: function(context) {
+    var getDepLiteralLines = function(controllerArg, nameIn) {
+      var con = {};
+      var startLine = 1000000;
+      controllerArg.forEach(function(Literal) {
+        var lineNo = Literal.loc.start.line;
+        if (startLine > lineNo) {
+          startLine = lineNo;
+        }
+        con[Literal[nameIn]] = lineNo - startLine;
+      });
+      return con;
+    };
+    return {
+      'CallExpression[callee.property.name=directive]': function(node) {
+        var arg = node.arguments;
+        // In angular, components function take 2 arguments and type of last
+        // arguments is an ArrayExpression.
+        if (arg.length !== 2 || arg[1].type !== 'ArrayExpression') {
+          return;
+        }
+        var lengthOfElements = arg[1].elements.length;
+        // Storing function() in functionNode by taking last
+        // element as because its type is FunctionExpression.
```

Comment:
No need for this comment. You can remove this :)

Reply:
Done!